src/Symfony/Component/HttpKernel/HttpCache/ResponseCacheStrategy.php

```diff
@@ -5,10 +5,6 @@
  *
  * (c) Fabien Potencier <fabien@symfony.com>
  *
- * This code is partially based on the Rack-Cache library by Ryan Tomayko,
- * which is released under the MIT license.
- * (based on commit 02d2b48d75bcb63cf1c0c7149c077ad256542801)
```

Comment:
is the rack-cache reference not true anymore? also, did you check if rack-cache refactored their thing to be more correct?

Reply:
The whole class is basically a rewrite, so I think this reference is obsolete. Also, I primarily changed this because fabbot.io said so. I did not check Rack-Cache.